.github/workflows/sandbox.yml

```diff
@@ -112,7 +49,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-single-buildx
       - name: Login to GitHub Container Registry
-        if: ${{ github.event_name == 'release' }}
+        if: ${{ github.event_name != 'release' }}
```

Comment:
This should change, right?

Reply:
this is correct bu github is not showing it

Reply:
https://github.com/flyteorg/flyte/blob/06532ae86cab41082950e7c1aec57107de23ff53/.github/workflows/sandbox.yml#L63